packages/KIcon/icons/icn-service-document.svg

```diff
@@ -0,0 +1 @@
+<svg width="12" height="10" viewBox="0 0 10 10" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M11.5 0H.5a.5.5 0 00-.5.5v1a.5.5 0 00.5.5h11a.5.5 0 00.5-.5v-1a.5.5 0 00-.5-.5zM3.5 4h-3a.5.5 0 00-.5.5v1a.5.5 0 00.5.5h3a.5.5 0 00.5-.5v-1a.5.5 0 00-.5-.5zM11.5 4h-6a.5.5 0 00-.5.5v1a.5.5 0 00.5.5h6a.5.5 0 00.5-.5v-1a.5.5 0 00-.5-.5zM11.5 8H.5a.5.5 0 00-.5.5v1a.5.5 0 00.5.5h11a.5.5 0 00.5-.5v-1a.5.5 0 00-.5-.5z" fill="#A3B6D9"/></svg>
```

Comment:
if the width is 12 shouldn't the viewBox be `0 0 12 10`?

Reply:
yeah i was looking at the docs and either `10` or `12` both looked good to me lol